src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -47,7 +58,7 @@
 
         .bcs-ActivityItem {
             position: relative;
-            box-shadow: 0 $container-padding 0 -#{$container-padding - 1} $bdl-gray-10;
+            box-shadow: 0 1px 0 0 $bdl-gray-10;
```

Comment:
what's the effect of changing this?

Reply:
can the `$container-padding` variable be removed now?

Reply:
I'll double check if it's used anywhere else and remove it if not.
This change makes the gray bars between the messages stretch to both edges of sidebar.

Reply:
Yea I'm worried that making these changes that are universal may negatively impact preview now. Preview does not currently support replies and some of these changes might break the experience. Has this been tested for preview?